### ISSUE TITLE
Investigate the Use of Vector Tiling

### ISSUE THREAD

**Issue by mataharimhairi** [hotosm/osm-export-tool#271]

**Comment by bdon** [comment]
I don't think this will have much benefit, if we're considering changing the slippy map to a vector-based renderer instead of OpenLayers. It will be compatible with less browsers, and most vector tilesets show less complete OSM information then the style we're using right now. Please re-open if you have additional thoughts @mataharimhairi or @mojodna

**Comment by mataharimhairi** [comment]
@bdon. the idea was to have a vector tiling slippy map that would react to the map feature selections of the user, so that they could physically see on the map what their selection (tree tag or yaml) was 'grabbing'. does this make sense?

**Comment by mataharimhairi** [comment]
Investigation work already started in this repository: https://github.com/hotosm/hot-qa-tiles
